src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -0,0 +1,50 @@
+﻿{
+  "title": "Application Insights for ASP.NET Core Web Applications",
+  "description": "Application Insights for ASP.NET Core web applications. See http://go.microsoft.com/fwlink/?LinkID=510752 for more information.",
```

Comment:
I think this is the correct link to point them to: https://azure.microsoft.com/en-us/documentation/articles/app-insights-asp-net-five/